net-nntp/sabnzbd/sabnzbd-4.1.0.ebuild

```diff
@@ -0,0 +1,168 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
can you update the PYTHON_COMPAT and re-test? pls check version 4.0.3

Reply:
That was deliberate. Officially upstream started supporting 3.12 in the 4.2.0 release cycle.

But I can test the older versions and see if they pass tests with 3.12....

Reply:
Update: I am unable to test 3.12 yet on my two AMD64 systems, there's too many packages still requiring 3.11. My minimal install with 3.12 on ARM64 is not compatible with all the test packages, so I cannot advertise that arch for this.

Someone else will need to test if they want 3.12 for the old releases. I will advertise 3.12 for 4.2.0 when it releases (should be soon, on RC2 now).

Reply:
I'm surprised you can't test it - you should be able to put */* PYTHON_TARGETS: python3_11 python3_12 in package.use?

Reply:
yeah I can do that, I forgot python supports multiple targets at once. Been building systems single target for years....

Reply:
Sorry for delay, passes 3.12 tests!